### DISCUSSION TITLE
Welcome to react-native-switch-selector Discussions!

### DISCUSSION THREAD

**Discussion by jkdrangel** [jkdrangel/react-native-switch-selector#3] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by tamilarasan061** [comment]
Hi @jkdrangel, Can the height of the selector be reduced below 40?
